README.rst

```diff
@@ -75,18 +75,20 @@ To use Octopus after installation
 3. Use octopus (it should be in the ``$PATH``). You can check the octopus version using ``octopus version``.
```

Comment:
In sections `[Compilation of Octopus using Spack](https://github.com/fangohr/octopus-in-spack/tree/split-off-docker-image#id6)` should we suggest users to use an environment and install octopus inside that?
Pros:
* Teaches users about its availability 
* Environments is good for isolation.
Cons:
* adds extra steps for users ( but its just a suggestion )

Reply:
Environments are a good idea. However, here I don't think this is helpful. Reasoning:

(i) if you only use spack to compile octopus, then you probably do not need an environment. 

(ii) if you use spack for more than octopus, and an environment would be good to use, then you probably know about it already.

Reply:
More documentation on how to use spack to compile Octopus is good. I think this should be on a separate page [i.e. in a different file] (to which we can point from the README here, and to which we can point from the Octopus documentation) so that it doesn't have all the material from the README that explains the purpose of the repo etc.

Please create a new PR for that.